src/Microsoft.Identity.Web.Certificate/ICertificateLoader.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Identity.Web
     public interface ICertificateLoader
     {
         /// <summary>
-        /// Load the certificate from the description, if needed.
+        /// Load the credentials from the description, if needed.
```

Comment:
>credentials [](http://example.com/codeflow?start=21&length=11)

why change the XML if this is still relating to certs?

Reply:
good catch. Thanks!